### ISSUE TITLE
Does not show more than one .should('contain') assertion when chained after .should('be.visible')

### ISSUE THREAD

**Issue by fsnopek** [cypress-io/cypress#16006]
### Current behavior
Test runner shows misleading info when more than one `.should('contain')` assertion is chained after `.should('be.visible')`
Consider following code

```js
 cy.get('[data-cy=mobile-payments-template]')
        .should('be.visible')
        .should('contain', 'Something')
        .should('contain', '+420 737 799 983');
```
This is displayed incorrectly in Test runner (notice the replaced `to contain Something` with `to be visible`)
![image](https://user-images.githubusercontent.com/36360797/114858971-6b1e7b80-9dea-11eb-92c3-a6978f323cd8.png)

When I delete the `should('be.visible')` assertion, it's displayed properly
![image](https://user-images.githubusercontent.com/36360797/114859101-9608cf80-9dea-11eb-9e06-2505ebea4a48.png)

### Desired behavior
Display `to contain Something`
![image](https://user-images.githubusercontent.com/36360797/114860480-56db7e00-9dec-11eb-848a-f5d29ba9291f.png)

### Versions
6.8.0

**Comment by sainthkh** [comment]
Confirmed that it doesn't work on 7.0.1.

**index.html**

```html
<html lang="en">
  <body>
    <div id="x">Something +123</div>
  </body>
</html>
```

**test.js**

```js
it('test', () => {
  cy.visit('index.html')

  cy.get('#x')
  .should('be.visible')
  .should('contain', 'Something')
  .should('contain', '+123')
})
```

**Comment by jennifer-shehane** [comment]
I went back to version 5.2.0, it looks like it's displayed this way for a while.

**Comment by jennifer-shehane** [comment]
Released in [`7.3.0`](https://on.cypress.io/changelog#7-3-0).

This comment thread has been locked. If you are still experiencing this issue after upgrading to
Cypress v7.3.0, please open a new issue.
